website/docs/r/dx_hosted_transit_virtual_interface_accepter.html.markdown

```diff
@@ -0,0 +1,102 @@
+---
+layout: "aws"
+page_title: "AWS: aws_dx_hosted_transit_virtual_interface_accepter"
+sidebar_current: "docs-aws-resource-dx-hosted-transit-virtual-interface-accepter"
+description: |-
+  Provides a resource to manage the accepter's side of a Direct Connect hosted transit virtual interface.
+---
+
+# Resource: aws_dx_hosted_transit_virtual_interface_accepter
+
+Provides a resource to manage the accepter's side of a Direct Connect hosted transit virtual interface.
+This resource accepts ownership of a transit virtual interface created by another AWS account.
+
+## Example Usage
+
+```hcl
+provider "aws" {
+  # Creator's credentials.
+}
+
+provider "aws" {
+  alias = "accepter"
+
+  # Accepter's credentials.
+}
+
+data "aws_caller_identity" "accepter" {
+  provider = "aws.accepter"
+}
+
+# Creator's side of the VIF
+resource "aws_dx_hosted_transit_virtual_interface" "creator" {
+  connection_id    = "dxcon-zzzzzzzz"
+  owner_account_id = "${data.aws_caller_identity.accepter.account_id}"
+
+  name           = "tf-transit-vif-example"
+  vlan           = 4094
+  address_family = "ipv4"
+  bgp_asn        = 65352
+
+  # The aws_dx_hosted_transit_virtual_interface
+  # must be destroyed before the aws_dx_gateway.
+  depends_on = ["aws_dx_gateway.example"]
+}
+
+# Accepter's side of the VIF.
+resource "aws_dx_gateway" "example" {
+  provider = "aws.accepter"
+
+  name            = "tf-dxg-example"
+  amazon_side_asn = 64512
+}
+
+resource "aws_dx_hosted_transit_virtual_interface_accepter" "accepter" {
+  provider             = "aws.accepter"
+  virtual_interface_id = "${aws_dx_hosted_transit_virtual_interface.creator.id}"
+  dx_gateway_id        = "${aws_dx_gateway.example.id}"
+
+  tags = {
+    Side = "Accepter"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `dx_gateway_id` - (Required) The ID of the [Direct Connect gateway](dx_gateway.html) to which to connect the virtual interface.
+* `virtual_interface_id` - (Required) The ID of the Direct Connect virtual interface to accept.
+* `tags` - (Optional) A mapping of tags to assign to the resource.
+
+### Removing `aws_dx_hosted_transit_virtual_interface_accepter` from your configuration
```

Comment:
This section should instead be a note at the top of the resource page, using the `~> **NOTE:** Message here` syntax

Reply:
Will do.